components/benefit_sponsors/app/controllers/benefit_sponsors/profiles/broker_agencies/broker_agency_profiles_controller.rb

```diff
@@ -26,34 +27,23 @@ class BrokerAgencyProfilesController < ::BenefitSponsors::ApplicationController
           "5" => "employer_profile.plan_years.start_on"
         }.freeze
 
-        def create
```

Comment:
It doesn't look like we're using it, but please confirm with @TreyE since he's the author of this. [commit](https://github.com/ideacrew/enroll/commit/a944aa821ea9c430759e61a141a9e67efdd873b3)